db/schema.rb

```diff
@@ -577,7 +578,6 @@
     t.boolean  "is_admin",               default: false
     t.string   "username"
     t.boolean  "is_disabled",            default: false
-    t.string   "token"
```

Comment:
Not sure what happened here, but we shouldn't be making this change. Can you please take a look?

Reply:
Sure, I will have a look at it.